src/Orc.CsvTextEditor/CsvTextEditorToolManagement/Interfaces/ICsvTextEditorToolManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ICsvTextEditorToolManager.cs" company="WildGums">
+//   Copyright (c) 2008 - 2016 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.CsvTextEditor.CsvTextEditorToolManagement
+{
+    using System.Collections.Generic;
+
+    public interface ICsvTextEditorToolManager
+    {
+        #region Properties
+        object Scope { get; }
```

Comment:
instead of using scopes here we could add ToolsManager property into CsvEditorService
much simplier